taskqueues.lua

```diff
@@ -201,6 +201,16 @@ function BuildWithLimitedNumber(tmpUnitName, minNumber)
 	end
 end
 
+function AmpOrGroundWeapon(MyTB)
```

Comment:
Please don't name the argument `MyTB`, which is already a global. The current convention is to name such arguments `tskqbhvr`, but anything would be less confusing than `MyTB`

Also, unless you have plans to use this function as part of a taskqueue, why put it in taskqueues.lua? If it's just used by taskqueuebehaviour, I suggest making it a local function or a class method in taskqueuebehaviour.lua. Then, any taskqueue function that needs to know if the factory should be amphibious can just use the behaviour's `AmpOrGroundWeapon` (either by passing the behaviour as an argument or the lazy way through `MyTB`).